components/add_users_to_team/index.js

```diff
@@ -17,7 +17,7 @@ import {isModalOpen} from 'selectors/views/modals';
 import AddUsersToTeam from './add_users_to_team.jsx';
 
 function mapStateToProps(state) {
-    const searchTerm = state.views.search.modalSearch;
+    const searchTerm = state.views.search.modalSearch && state.views.search.modalSearch.trim();
```

Comment:
Probably better to just trim it in the action or reducer before it's stored, so you should only have to trim once rather than everywhere `state.views.search.modalSearch` is used

Reply:
done